README.md

```diff
@@ -26,14 +27,48 @@ jobs:
   - put: <resource name>
 ```
 
-See [the instructions for getting your AWS credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html#cli-signup), then pass them and the bucket name in as [parameters](http://concourse.ci/fly-set-pipeline.html#section_parameters).
+### Options
+The `options` parameter is synonymous with the options that `aws cli` accepts for `sync`.
+Please see
+[S3 Sync Options](http://docs.aws.amazon.com/cli/latest/reference/s3/sync.html#options)
+and pay special attention to the
+[Use of Exclude and Include Filters](http://docs.aws.amazon.com/cli/latest/reference/s3/index.html#use-of-exclude-and-include-filters)
```

Comment:
Do you feel strongly about hard line wrapping? I feel pretty strongly in the other direction :wink: Also, missing a period at the end of the sentence.